Antidote/DialingCallViewController.m

```diff
@@ -39,6 +39,28 @@ - (void)viewDidLoad
     [self installConstraints];
 }
 
+- (void)viewDidLayoutSubviews
+{
+    [self updateFriendAvatar];
+}
```

Comment:
Now I'm thinking if we should call `updateFriendAvatar` method here. Afaik `viewDidLayoutSubviews` is called quite a lot of times. In case if we need to update it here it is better to add following check to `updateFriendAvatar` method

```
if (self.friendAvatar.frame.size == diameterOfImage) {
   return;
}
```